ross/fluid_flow/fluid_flow.py

```diff
@@ -0,0 +1,400 @@
+
+    def calculate_pressure_matrix(self):
+        """This function calculates the pressure matrix
+        """
+        if self.bearing_seal_type == 0:
+            for i in range(self.nz):
+                for j in range(self.ntheta):
+                    self.p_mat[i][j] = ((-3*self.visc*self.omega)/self.difference_between_radius**2)*\
+                                       ((i*self.dz)**2 - (self.length**2)/4)*\
+                                       (self.eccentricity_ratio*np.sin(j*self.dtheta))/\
+                                       (1 + self.eccentricity_ratio*np.sin(j*self.dtheta))**3
+            self.pressure_matrix_available = True
+        return self.p_mat
+
+    def calculate_coefficients(self, plot_cut=False):
+        """This function calculates the constants that form the Poisson equation
+        of the discrete pressure (central differences in the second
+        derivatives). It is executed when the class is instantiated.
+        Parameters
+        ----------
+        plot_cut: bool
+            If True, plots a cut at z=0 and show, so the user can check the
+            internal and external radius as well as eccentricity.
+        """
+        if self.length/self.radius_stator <= 1/8:
+            self.bearing_seal_type = 0
+        elif self.length/self.radius_stator > 4:
+            self.bearing_seal_type = 1
+        else:
+            self.bearing_seal_type = 2
+        if plot_cut:
+            plt.figure(self.plot_counter)
+            self.plot_counter += 1
+        for i in range(self.nz):
+            zno = i * self.dz
+            self.z[0][i] = zno
+            for j in range(self.ntheta):
+                gama = j * self.dtheta
+                [radius_external, self.xre[i][j], self.yre[i][j]] =\
+                    self.external_radius_function(gama)
+                [radius_internal, self.xri[i][j], self.yri[i][j]] =\
+                    self.internal_radius_function(zno, gama)
+
+                if plot_cut:
+                    # Plot a cut at z=0:
+                    if i == 0:
+                        plt.plot(self.xre[i][j], self.yre[i][j], 'r.')
+                        plt.plot(self.xri[i][j], self.yri[i][j], 'b.')
+                        plt.plot(0, 0, '*')
+                        plt.title('Cut in plane Z=0')
+                        plt.xlabel('X axis')
+                        plt.ylabel('Y axis')
+                        plt.axis('equal')
+
+                self.re[i][j] = radius_external
+
+                self.ri[i][j] = radius_internal
+        if plot_cut:
+            plt.show()
+            plt.close('all')
+
+    def internal_radius_function(self, z, gama):
+        """This function calculates the radius of the rotor given the
+        radius and the position z.
+        Parameters
+        ----------
+        z: float
+            Distance along the z-axis.
+        gama: float
+            Gama is the distance in the theta-axis. It should range from 0 to 2*np.pi.
```

Comment:
Add the following information to the docstring:
```python
"""This function calculates the radius of the rotor given the
...
    Returns
    -------
    radius_external: float
        <info>
    xre: float
        <info>
    yre: float
        <info>
```

Reply:
Done.